CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ In order to review the effects of your changes, you should create your own Kuber
 In case you make changes to the Connaisseur Docker image itself or code for that matter, you need to re-build the image and install it locally for testing. This requires a few steps:
 
 1. In `helm/values.yaml`, set `imagePullPolicy` to `IfNotPresent`.
-2. Configure your local environment to use the Kubernetes docker daemen. In Minikube, this can be done via `eval (minikube docker-env)`.
+2. Configure your local environment to use the Kubernetes docker daemon. In Minikube, this can be done via `eval (minikube docker-env)`.
```

Comment:
The integration test makes the following lines obsolete doesn't it? https://github.com/sse-secure-systems/connaisseur/pull/42/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R71

We could mention in the Contributing Guidelines that we encourage to add integration testing for new features maybe

Reply:
Link doesnt work for me. Found some lines you might've meant though. Can you check whether new changes work for you?

Reply:
Exactly! Thanks and sorry for the broken link!